### DISCUSSION TITLE
support for romanized Indian languages?

### DISCUSSION THREAD

**Discussion by tfriedel** [pemistahl/lingua-py#183] [Show and tell]
Hi Peter!
Have you tried your library with romanized versions of Indian languages like Hindi? 
There's a new library that's better at this than for example fasttext:
https://github.com/AI4Bharat/IndicLID 

I've noticed a lot Indian text for example in chat apps is romanized and usually not supported well by current language identification libraries.

**Comment by pemistahl** [comment]
Hi Thomas, thanks for letting me know about this library. I didn't know that Indian languages are used more and more in romanized versions. It's surely a good idea to support those transliterations in Lingua. I will put it on my bucket list. Viele Grüße nach Berlin! :-)
